email.rst

```diff
@@ -4,102 +4,57 @@
 How to Send an Email
 ====================
 
-Sending emails is a classic task for any web application and one that has
-special complications and potential pitfalls. Instead of recreating the wheel,
-one solution to send emails is to use the SwiftmailerBundle, which leverages
-the power of the `Swift Mailer`_ library. This bundle comes with the Symfony
-Standard Edition.
+Sending emails is a common task for any web application and one that has
+special complications and potential pitfalls. Symfony provides a mailer feature
```

Comment:
I think the first sentence is crap... and it's not *your* change - I think the old sentence (I probably wrote it?) is crap. Let's remove it entirely.